src/controllers/subject.js

```
@@ -28,7 +28,21 @@ const getSubjectById = async (req, res) => {
   res.status(200).json(subject);
 };
 
+const updateSubject = async (req, res) => {
+  const { id } = req.params;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)